src/Radicle/Daemon/Common.hs

```diff
@@ -0,0 +1,66 @@
+module Radicle.Daemon.Common
```

Comment:
Can you add documentation to the values and types defined here that don’t yet have documentation?